### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by devos50**
This PR fixes various issues related to the `AnydexModule` class, which loads Gumby without Tribler.

### CODE REVIEW

File: gumby/modules/tribler_module.py [hunk 1]
```diff
@@ -262,8 +262,8 @@ def write_overlay_statistics(self):
 
         # Write bandwidth statistics
         with open('bandwidth.txt', 'w') as bandwidth_file:
-            bandwidth_file.write("%d,%d" % (self.session.ipv8.endpoint.bytes_up,
-                                            self.session.ipv8.endpoint.bytes_down))
+            bandwidth_file.write("%d,%d" % (self.session.ipv8.endpoint.interfaces["UDPIPv4"].bytes_up,
```
Comment (qstokkink) [inline gumby/modules/tribler_module.py]: Hmm.. this is not pretty. I'll add the `bytes_up` and `bytes_down` properties to the `DispatcherEndpoint`.

Comment (qstokkink) [reply gumby/modules/tribler_module.py]: If you want to wait for https://github.com/Tribler/py-ipv8/pull/837, you don't need this change.

Comment (qstokkink) [reply gumby/modules/tribler_module.py]: If you use the latest IPv8 you can 
~~drop commit d6df84dbca2d65f74fd9a879e3d46b9bf692f28d from this PR~~, or rather edit it, it seems -- that contains a bit exta.

Comment (devos50) [reply gumby/modules/tribler_module.py]: Fixed, thanks for the addition! 👍

**Review by qstokkink — APPROVED** [review]

File: gumby/modules/tribler_module.py (hunk 1 above)
Comment (qstokkink) [reply gumby/modules/tribler_module.py]: If at all possible I'd like IPv8 to good looking, complete and easy to use 😄

**Pull request merged**